roles/confluent.zookeeper/defaults/main.yml

```diff
@@ -39,3 +39,7 @@ zookeeper_service_unit_overrides:
 
 ### Time in seconds to wait before starting Zookeeper Health Checks.
 zookeeper_health_check_delay: 5
+
+### Group names to use when gathering dynamic groups
```

Comment:
please do the same annotation thing here and run `python doc.py` to add to the VARIABLES.md file

Reply:
Done